testing/tests/DevExpress.ui.widgets.dataGrid/editing.tests.js

```diff
@@ -9299,6 +9298,21 @@ QUnit.test("When showing dxDataGrid in detail, 'select all' function of master g
     assert.equal(testElement.find('.' + rowSelectionClass).length, 6, "7 rows - 1 row that edit, 2 detail row doesn't selected");
 });
 
+//T579296
+QUnit.test("cancel edit row after expand row", function(assert) {
+    var testElement = $('#container');
+
+    this.rowsView.render(testElement);
+
+    this.editingController.editRow(2);
+
+    assert.strictEqual(this.editingController._editRowIndex, 2);
```

Comment:
I think that use public method isEditing is better than private field_editRowIndex.